src/KeyBindingsDefaults.ts

```diff
@@ -411,10 +412,24 @@ const navigationBindings = (): KeyBinding<NavigationAction>[] => {
     ];
 };
 
+const labsBindings = (): KeyBinding<LabsAction>[] => {
+    return [
+        {
+            action: LabsAction.ToggleHiddenEventVisibility,
+            keyCombo: {
+                key: Key.H,
+                ctrlOrCmd: true,
+                shiftKey: true,
+            },
+        },
+    ];
```

Comment:
I believe this should return an empty array, if `SdkConfig.get()['showLabsSettings']` is false